components/apimgt/org.wso2.carbon.apimgt.persistence/src/main/java/org/wso2/carbon/apimgt/persistence/APIPersistence.java

```diff
@@ -255,6 +255,16 @@ DevPortalContentSearchResult searchContentForDevPortal(Organization org, String
     /* ==== Async API Definition ==========
     ============================================= */
 
+    /**
+     * Save OAS Schema definition
```

Comment:
Change to 'Save Async API definition'

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/10621/commits/d5abe94f46e9e929b22ad4e684ae4d9122edc925